src/brain/issueLabelHandler/followups.ts

```diff
@@ -158,5 +169,25 @@ export async function ensureOneWaitingForLabel({
 
   await modifyProjectIssueField(itemId, labelName, STATUS_FIELD_ID, octokit);
 
+  let timeToRespondBy;
+  if (labelName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
+    timeToRespondBy =
+      (await calculateSLOViolationTriage(UNTRIAGED_LABEL, issue.labels)) ||
+      moment().toISOString();
+  } else if (labelName === WAITING_FOR_SUPPORT_LABEL) {
+    timeToRespondBy =
+      (await calculateSLOViolationRoute(UNROUTED_LABEL)) ||
+      moment().toISOString();
+  } else {
```

Comment:
Should we care about these missing tests?

Reply:
wrote some tests to cover those edge cases